### DISCUSSION TITLE
Editor throwing exceptions when Ctrl+Z

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#16416] [Q&A] [answered]
Steps to reproduce:
1) select a random map (as long as it contains sliders) and right-click edit.
2) choose a slider, Ctrl+C, scroll to somewhere different in the timeline, Ctrl+V
3) move the slider, drag the control points of the slider
4) undo (Ctrl+Z)
5) note that osu! will throw exceptions like this: `Failed to process line "Infinity,-1000,4,2,0,55,0,0" into "beatmap name": Value is too high` the numbers in the exception may vary.

Unluckily I can't 100% reproduce this (even on the same map), but (at least for me) the chance of seeing it is pretty high; the undo seems to succeed anyway, though idk if it has other side-effects on the map
![exception](https://user-images.githubusercontent.com/33959010/149010174-23f9b8f7-d8e7-4602-808f-2479cd6e8b25.png)

[runtime.log](https://github.com/ppy/osu/files/7849507/runtime.log)

osu!lazer 2022.108.0, win10 20H2

**Comment by bdach** [comment, marked as answer]
#16417
